ATP.BusinessLogicLayer/GenericRepo/FlightRepository.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ATP.DataAccessLayer.IGenericRepo;
+using ATP.DataAccessLayer.Models;
+using CsvHelper;
+using System.Globalization;
+
+namespace ATP.BusinessLogicLayer.GenericRepo;
+public class FlightRepository : IGenericRepo<Flight>
+{ 
+    private List<Flight> _flights;
+
+    public FlightRepository(string csvFilePath)
+    {
+        _flights = LoadFlightsFromCsv(csvFilePath);
+    }
+    private List<Flight> LoadFlightsFromCsv(string csvFilePath)
+    {
+        List<Flight> flights = new List<Flight>();
+
+        using (var reader = new StreamReader(csvFilePath))
+        using (var csv = new CsvReader(reader, CultureInfo.InvariantCulture))
+        {
+            flights = csv.GetRecords<Flight>().ToList();
+        }
```

Comment:
You can use the keyword using in more clear way:
```suggestion
        using var reader = new StreamReader(csvFilePath);
        using var csv = new CsvReader(reader, CultureInfo.InvariantCulture);
        flights = csv.GetRecords<Flight>().ToList();
```